grails-app/views/notify/requirements.gsp

```diff
@@ -25,11 +25,16 @@
         </ul>
     </div>
 <p>
-    If you have any questions or concerns, please contact our colleague who created this Consent Group and is cc’d on this email.
+    A member from the OSAP team, cc’d here will be in touch with you shortly to proceed with obtaining them.
 </p>
+<g:if test="${values.get("type").equals("Project") || values.get("feeForService")?.equals("true") || values.get("areSamplesComingFromEEAA")?.equals("true") || values.get("isConsentUnambiguous")?.equals("true")}">
```

Comment:
`"Project"` here should also be an enum for safer refactoring/typing. Here is one way to do that: https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/#enumerations-with-objects

Reply:
This case is a template to render the email content. I don't think that is good idea to add js to this. Maybe the best idea is to have an enum with two values Project and CG and use it here and the previous case.

Reply:
No. I tried to solve this with an enum in the template but it didn't work. Can we merge this any way?